merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/PostgresResultsCellRepository.java

```diff
@@ -420,15 +421,14 @@ private static void postActivities(
               e -> unfinishedActivityToRecord(e.getValue())));
       allActivityRecords.putAll(simulatedActivityRecords);
 
-      final var simIdToPgId = postActivitiesAction.apply(
+      postActivitiesAction.apply(
           datasetId,
           allActivityRecords,
           simulationStart);
 
       updateSimulatedActivityParentsAction.apply(
           datasetId,
-          allActivityRecords,
-          simIdToPgId);
+          allActivityRecords);
```

Comment:
Once we update the `PostSpansAction` to set the parent id, we can delete the update action altogether

Reply:
I removed the function